### ISSUE TITLE
IP Groups: Cards for dataverse, dataset, files do not appear after access granted to group.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#3105]
Related to #1380 
While testing found that now that we can grant access to dataverses and see them via a direct link, the card does not appear when browsing or in My Data. Cards for datasets and files also do not appear but may currently be blocked by permissions not allowing direct link access in #1380.

**Comment by kcondon** [comment]
This might be a duplicate of #1513

**Comment by pdurbin** [comment]
Today @michbarsinai @scolapasta @kcondon and I discussed IP groups and I've been asked to try to reproduce this bug in the branch that pull request #3103 is based on. It's believed this is still a bug.

**Comment by kcondon** [comment]
This is working now. Phil we can close this unless there is something more you want to do.

**Comment by pdurbin** [comment]
@kcondon awesome. So the fix is in pull request #3103 at least partially fixed by e6438c7. I'll close this issue but I'll note that #3273 is still open which is about "groups within groups".
